.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -458,6 +458,14 @@ jobs:
           cmake --build . --config Release --parallel 4
           ./hello
 
+      - name: Build Sample [simple]
+        if: ${{ matrix.light == 'green' }}
+        run: |
+          ./build.sh -p host-interp
+          python3 ./sample_test_run.py $(pwd)/out
+          exit $?
+        working-directory: ./samples/simple
```

Comment:
Should be `./out/simple -s`? `-s` is to start app manager in server mode. But a question is that it never exits?

Reply:
we are using a python script to handle that:
- https://github.com/lum1n0us/wasm-micro-runtime/blob/release_wamr_sdk/samples/simple/sample_test_run.py#L20 starts the app manager in server mode.
- https://github.com/lum1n0us/wasm-micro-runtime/blob/release_wamr_sdk/samples/simple/sample_test_run.py#L95 always closes the app manager

Reply:
Good, cool